examples/capturing-smartbanner/above-tag.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
```

Comment:
Typically the `examples` folder is where all of the code for demonstrations go. Could we move these to the `tests/fixtures` folder?